src/components/render_movie_card.js

```diff
@@ -1,38 +1,51 @@
 import React from 'react';
 import noImage from '../images/no_image_available.jpeg';
+import { Link } from 'react-router-dom';
 
 export default function (props) {
 
-  const { movie, genres } = props;
-
+  const { movie, genres, quality } = props;
+  // to={`/movie/${movie.id}`}
   return (
     <div className="col-md-3 col-lg-3">
-      <div className="detail-movie">
-        <div className="detail-img">
-          <img src={renderMovieImg(movie)} alt='' />
-          <div className="hidden-details">
-            <div><i className="fas fa-star"></i></div>
-            <div className="hidden-details-rating">{movie.rating} / 10</div>
-            <div className="hidden-details-genre">{renderMovieGenre(genres)}</div>
-            <div className="hidden-details-btn">View Details</div>
+      <Link to={`/movie/${movie.id}`} >
+        <div className="detail-movie">
+          {renderMovieQuality(quality)}
+          <div className="detail-img">
+            <img src={renderMovieImg(movie)} alt='' />
+            <div className="hidden-details">
+              <div><i className="fas fa-star"></i></div>
+              <div className="hidden-details-rating">{movie.rating} / 10</div>
+              <div className="hidden-details-genre">{renderMovieGenre(genres)}</div>
+              <div className="hidden-details-btn">View Details</div>
+            </div>
           </div>
+          <div className="detail-title">{movie.title}</div>
+          <div className="detail-movieYear">{movie.year}</div>
         </div>
-        <div className="detail-title">{movie.title}</div>
-        <div className="detail-movieYear">{movie.year}</div>
-      </div>
+      </Link>
     </div> 
 );
-}
+};
+
+function renderMovieQuality(quality) {
```

Comment:
quality always comes as a string eg. '1080p' or '720p'? Or quality can also be an array of those values?